sdcm/k8s_configs/ingress-controller/10_haproxy-kubernetes-ingress.svc.yaml

```diff
@@ -3,6 +3,9 @@ kind: Service
 metadata:
   name: haproxy-kubernetes-ingress
   namespace: haproxy-controller
+  annotations:
```

Comment:
Adding the link to the docs you mention in the task for this PR would be very useful:

https://kubernetes-sigs.github.io/aws-load-balancer-controller/v2.4/guide/service/annotations/#aws-resource-tags

Reply:
added